back-end/create-database-sql

```diff
@@ -0,0 +1,3 @@
+CREATE DATABASE IF NOT EXISTS slothstagram ;
+CREATE USER slothstagram_admin WITH PASSWORD 'KristenBell';
+GRANT ALL PRIVILEGES ON DATABASE  TO slothstagram_admin;
```

Comment:
I believe "IF NOT EXISTS" is not valid PostgreSQL: https://stackoverflow.com/questions/18389124/simulate-create-database-if-not-exists-for-postgresql

You also don't mention the database name in the "GRANT" query.